src/Core.pm

```diff
@@ -1651,20 +1678,31 @@ sub RunCommand
   my $command = shift;
   my $try = shift;
 
+  my $lang = $self->GetSysconfigVar('language', 'RC_LANG');
```

Comment:
I thought about that, too. But that would add a new dependency of Core.pm on Tools.pm. Also, I didn't see any use of GetSystemLanguage() at least within yast.

Given that this is dead code in maintenance mode I decided to go for the smallest change.

Note that this was implemented in master in a completely different way in https://github.com/openSUSE/perl-bootloader/pull/104.